### PR TITLE
Documentation for k8s-policy #85

### PR DESCRIPTION
**Pull request by caseydavenport**
Documentation for https://github.com/projectcalico/k8s-policy/pull/85

Do not merge until 85 is merged.

### CODE REVIEW

**Comment by neiljerram** [comment]
Would like to check my understanding here...  I think that:

$NO_MATCH_ORDER is used as the order on a Calico policy that just says 'pass' - whose effect is that the packet is allowed or denied according to the namespace's isolation annotation.

Some order value A (what?) is used as the order on each Calico policy that is generated from a NetworkPolicy resource.

IF the user also manually programs OTHER Calico policies, with order B, they need to ensure that B < $NO_MATCH_ORDER.  So if they are doing enough of such programming, that's why they might want to set NO_MATCH_ORDER larger than its default.

Is that right?

Do we also need to say something about NO_MATCH_ORDER > A, and possibly allow A to be configurable?

**Comment by caseydavenport** [comment]
@neiljerram ready for another look.  I've tweaked the wording so that I think it's more useful for users.  Thanks!

File: master/reference/policy-controller/configuration.md [hunk 1]
```diff
@@ -47,20 +47,21 @@ It is recommended to use the following configuration for API access:
...
 ### Kubernetes API access
 
-The leader election container also needs Kubernetes API access, which can be configured through a `kubeconfig` file placed in 
+The leader election container also needs Kubernetes API access, which can be configured through a `kubeconfig` file placed in
 the root directory of the container. This can be done by mounting a file from the host, or using Kubernetes [ConfigMap resources](http://kubernetes.io/docs/user-guide/configmap/).
 
 ### Other configuration
 
 * `LOG_LEVEL`: Supports the standard Python log levels. e.g. `LOG_LEVEL=debug`, defaults to `info`
+* `NO_MATCH_ORDER`: The order used for "no match" policies which implement default behavior.  New policies should not be created with an order greater than this value or else they will be ignored.  Default: 2000
```
Comment (neiljerram) [inline master/reference/policy-controller/configuration.md]: Is "New" significant here?  I don't think so.  In other words, I believe that any existing policies with order > NO_MATCH_ORDER would also be ignored.

May I suggest:
```
* `NO_MATCH_ORDER`: The order used for the "no match" policies which implement default behavior.  Policies with an order greater than this value will be ignored.  Default: 2000
```

**Comment by caseydavenport** [comment]
Going to close this, because the `no-match` policy is actually going away in v2.4 due to changes required for GA NP support.

**Pull request closed without merging**